donate/settings/environment.py

```diff
@@ -0,0 +1,113 @@
+import os
+import environ
+
+CSP_DEFAULT_SRC = ("'self'")
+CSP_SCRIPT_SRC = ("'self'", "'unsafe-inline'", "'unsafe-eval'",
```

Comment:
Point of concern: do we _need_ `unsafe-inline` and `unsafe-eval`? They're kind of... unsafe

Reply:
I checked as part of the VA and we determined that we do need at least one of them I'll check which it is.

Reply:
Based on my tests, we must use unsafe-inline because of Paypal. We can remove unsafe-eval.